public/index.html

```diff
@@ -21,6 +22,8 @@
     <title></title>
     <meta name="keywords" content="Personal Assistant Free Software Artificial Intelligence Chat Bot Machine Learning Scientific User Support IoT"/>
     <meta name="description" content="Chat with an Open Source Personal Assistant"/>
+    <meta name="theme-color" content="#4285f4">
+    <meta http-equiv="Cache-Control" content="max-age=2628002">
```

Comment:
This approach would let the cache expire only after a month, and we have deployments made almost daily. So, the latest JS bundle wouldn't be installed as, it would still use it from cache. We need to add some config which changes the cache name on every deployment.

Reply:
We can make it update after a day. But I think there can't be some approach to change cache name on every deployment.

Reply:
But if we cache only ``build/static/css/**.css`` and ``build/static/js/**.js`` then it will only cache styles and JavaScript instead of whole app. Can I do this?

Reply:
@MuhammadWasif The bundled files also goes into the static folder.

Reply:
So, can we make it update after a day? I think that will be a good approach.